docs/learn/learn-accounts.md

```diff
@@ -489,10 +491,12 @@ console.log("DepositFactor", deposit(0, 32));
 
 Thus the deposit values can be calculated as shown in the table below.
 
-|               | Polkadot (DOT) | Kusama (KSM)   | Polkadot (planck) | Kusama (planck) |
-| ------------- | -------------- | -------------- | ----------------- | --------------- |
-| DepositBase   | 20.088         | 3.3401         | 200880000000      | 3340100000000   |
-| DepositFactor | .032           | 0.005333333312 | 320000000         | 5333333312      |
+|          | Deposit Base                                                                                                    | Deposit Factor                                                                                                  |
+| -------- | --------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| Polkadot | <RPC network="polkadot" path="consts.multisig.depositBase" defaultValue={200880000000} filter="humanReadable"/> | <RPC network="polkadot" path="consts.multisig.depositFactor" defaultValue={320000000 } filter="humanReadable"/> |
+| Polkadot | <RPC network="polkadot" path="consts.multisig.depositBase" defaultValue={200880000000}/> Planck                 | <RPC network="polkadot" path="consts.multisig.depositFactor" defaultValue={320000000 }/> Planck                 |
+| Kusama   | <RPC network="kusama" path="consts.multisig.depositBase" defaultValue={66959996400} filter="humanReadable"/>    | <RPC network="kusama" path="consts.multisig.depositFactor" defaultValue={106665600} filter="humanReadable"/>    |
+| Kusama   | <RPC network="kusama" path="consts.multisig.depositBase" defaultValue={66959996400}/> Planck                    | <RPC network="kusama" path="consts.multisig.depositFactor" defaultValue={106665600}/> Planck                    |
```

Comment:
This formatting is a bit overwhelming to look at but doesn't get reformatted by prettier which is good.  In the past we noticed you can't put tables in conditional rendering  so we are pretty much stuck with something that looks like this unless we create a custom react component just for this small table which seems like overkill.

@DrW3RK If you think the long line formatting is not adequate I can break it out to a custom component, let me know how you feel about it.  I also chose to transpose the table because the Deposit Base/Factor are the focus and the RPC filters provide the units automatically so I thought it look cleaner but it can also be reversed back if desired.